Marlin/ultralcd_impl_DOGM.h

```diff
@@ -749,11 +757,12 @@ static void lcd_implementation_status_screen() {
   #define lcd_implementation_drawmenu_setting_edit_callback_bool(sel, row, pstr, pstr2, data, callback) lcd_implementation_drawmenu_setting_edit_generic_P(sel, row, pstr, (*(data))?PSTR(MSG_ON):PSTR(MSG_OFF))
 
   void lcd_implementation_drawedit(const char* const pstr, const char* const value=NULL) {
-    const uint8_t labellen = lcd_strlen_P(pstr), vallen = lcd_strlen(value);
-    uint8_t lcd_width, char_width,
-            rows = (labellen > LCD_WIDTH - 2 - vallen) ? 2 : 1;
+    const uint8_t labellen = lcd_strlen_P(pstr),
+                  vallen = lcd_strlen(value),
+                  rows = (labellen > LCD_WIDTH - 2 - vallen) ? 2 : 1;
 
     #if ENABLED(USE_BIG_EDIT_FONT)
+      uint8_t lcd_width, char_width;
       if (labellen <= LCD_WIDTH_EDIT - 1) {
         if (labellen >= LCD_WIDTH_EDIT - vallen) rows = 2;
```

Comment:
I got a compilation error.
<details>
<summary>error message</summary>

```
In file included from \AppData\Local\Temp\arduino_build_642648\sketch\ultralcd.cpp:56:0:

\AppData\Local\Temp\arduino_build_642648\sketch\ultralcd_impl_DOGM.h: In function 'void lcd_implementation_drawedit(const char*, const char*)':

ultralcd_impl_DOGM.h:767: error: assignment of read-only variable 'rows'

         if (labellen >= LCD_WIDTH_EDIT - vallen) rows = 2;

                                                       ^

exit status 1
assignment of read-only variable 'rows'
```
</details>

When `USE_BIG_EDIT_FONT` is enabled, `row` must not be `const`.